_inc/lib/xmlrpc-async-call.php

```diff
@@ -0,0 +1,131 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
If we're doing a new file, can we name it to comply with WPCS?

Reply:
The decision here was to name it in a way it is usable with `jetpack_require_lib`. It looks for the name of the file and it would be not really handy to have to type the whole name in the call. Do we have any better pattern for that? What do you think?

Reply:
If I move it to the connection package we won't have this problem